collectors/ebpf.plugin/ebpf.c

```diff
@@ -483,7 +483,17 @@ static void ebpf_exit()
     if (unlink(filename))
         error("Cannot remove PID file %s", filename);
 
-    ebpf_exit_plugin = 2;
+#ifdef NETDATA_INTERNAL_CHECKS
+    error("Good bye world! I was PID %d", main_thread_id);
+#endif
+    printf("DISABLE\n");
+
+    if (!shm_ebpf_cgroup.header) {
+        munmap(shm_ebpf_cgroup.header, shm_ebpf_cgroup.header->body_length);
```

Comment:
Shouldn't this be the opposite? If you do have a valid `shm_ebpf_cgroup.header` to unmap it?

Reply:
You're right, I inverted the calls, thank you!

Reply:
I meant the `if (!shm_ebpf_cgroup.header)` statement :smiley:  With this check, you will try to munmap a null pointer

Reply:
Thanks, I fixed.